components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/public/js/throttling/advanced-throttling/index.js

```diff
@@ -1,38 +1,35 @@
 $(function () {
-    var swaggerClient = new SwaggerClient({
-        url: swaggerURL,
-        success: function (swaggerData) {
-            setAuthHeader(swaggerClient);
-            swaggerClient["Throttling Tier (Collection)"].get_policies_tierLevel({"tierLevel":"api"},
-                                                                       {"responseContentType": 'application/json'},
-            function (jsonData) {
-                var raw_data = {
-                    data: jsonData.obj
-                };
-                var callbacks = {
-                    onSuccess: function () {
-                        _initDataTable(raw_data);
-                    },
-                    onFailure: function (message, e) {
+    var policyInstance = new Policy();
 
-                    }
-                };
-                var mode = "OVERWRITE";
-                var obj = {};
-                obj.list=jsonData.obj;
-                UUFClient.renderFragment("org.wso2.carbon.apimgt.web.admin.feature.policy-view", obj,
-                                         "policy-view", mode, callbacks);
-            }, function (error) {
-                        if (error.status == 401) {
-                            redirectToLogin(contextPath);
-                        }
-                    }
-            );
-        },
-        failure: function (error) {
-            console.log("Error occurred while loading swagger definition");
-        }
+    var promised_create =  policyInstance.getAllPoliciesByTier("api");
```

Comment:
This is a get operation, so better to have a different variable name (promised_get_tiers). Change in all the places